lib/workers/repository/process/index.ts

```diff
@@ -121,14 +121,17 @@ export async function extractDependencies(
     branchList: [],
     packageFiles: {},
   };
-  if (GlobalConfig.get('platform') !== 'local' && config.baseBranches?.length) {
-    config.baseBranches = unfoldBaseBranches(
+  if (
+    GlobalConfig.get('platform') !== 'local' &&
+    config.baseBranchPatterns?.length
+  ) {
+    config.baseBranchPatterns = unfoldBaseBranches(
```

Comment:
```suggestion
    config.baseBranches = unfoldBaseBranches(
```

Shouldn't we use the exisiting name here, so we can later distinguish betrween both values?
Otherwise this information is gone when branchify updates